blocklyc.html

```diff
@@ -886,59 +533,62 @@ <h4><span class="keyed-lang-string" data-key="client_chrome_run_title"></span></
-                            <svg preserveAspectRatio="xMinYMin" xmlns="http://www.w3.org/2000/svg" width="15" height="15"><path d="M7,8 L8,8 8,11 8,11 7,11 Z" style="stroke-width:1px;stroke:#8a6d3b;fill:none;"/><circle cx="7.5" cy="7.5" r="6" style="stroke-width:1.3px;stroke:#8a6d3b;fill:none;"/><circle cx="7.5" cy="5" r="1.25" style="stroke-width:0;fill:#8a6d3b;"/></svg>
+                            <span class="bpIcon" data-icon="warningCircle"></span>&nbsp;
                             <span class="keyed-lang-string" data-key="client_update_warning"></span>
                         </div>
                         <div id="client-unknown-span" class="alert alert-danger bpc-version hidden">
-                            <svg preserveAspectRatio="xMinYMin" xmlns="http://www.w3.org/2000/svg" width="15" height="15"><path d="M1,12 L2,13 13,13 14,12 8,2 7,2 1,12 Z M7.25,6 L7.75,6 7.5,9 Z" style="stroke-width:1.5px;stroke:#a94442;fill:none;"/><circle cx="7.5" cy="10.75" r="1" style="stroke-width:0;fill:#a94442;"/><circle cx="7.5" cy="5.5" r="1" style="stroke-width:0;fill:#a94442;"/></svg>
+                            <span class="bpIcon" data-icon="dangerTriangle"></span>&nbsp;
                             <span class="keyed-lang-string" data-key="client_unknown"></span>
                         </div>
                         <div id="client-danger-span" class="alert alert-danger bpc-version hidden">
-                            <svg preserveAspectRatio="xMinYMin" xmlns="http://www.w3.org/2000/svg" width="15" height="15"><path d="M1,12 L2,13 13,13 14,12 8,2 7,2 1,12 Z M7.25,6 L7.75,6 7.5,9 Z" style="stroke-width:1.5px;stroke:#a94442;fill:none;"/><circle cx="7.5" cy="10.75" r="1" style="stroke-width:0;fill:#a94442;"/><circle cx="7.5" cy="5.5" r="1" style="stroke-width:0;fill:#a94442;"/></svg>
+                            <span class="bpIcon" data-icon="dangerTriangle"></span>&nbsp;
                             <span class="keyed-lang-string" data-key="client_update_danger"></span>
                         </div>
                         <hr />
                         
                         <div class="clients" id="client-instructions-original">
                             <!-- MacOS client -->
                             <div class="client-instructions MacOS">
-                                <h4><span class="keyed-lang-string" data-key="clientdownload_download_installer"></span></h4>
+                                <h4 class="keyed-lang-string" data-key="clientdownload_download_installer">&nbsp;</h4>
                             </div>
                             <div class="client MacOS">
-                                <img src="//:0" data-src="images/os-icons/mac_os.png" alt="Mac OS icon"/>
+                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="images/os-icons/mac_os.png" alt="Mac OS icon"/>
```

Comment:
How is the base64 string generated?

Reply:
I found it by googling "smallest valid image".  It's a 1px by 1px GIF.